Toolset/libraries/revideextensionlibrary.livecodescript

```diff
@@ -668,6 +668,68 @@ private function __fetchModuleData pExtensionFile
    return tDataA
 end __fetchModuleData
 
+private function __extensionIsBuiltin pID
+   set the itemdelimiter to "."
+   return the number of items in pID is 3
+end __extensionIsBuiltin
+
```

Comment:
Is this viable when custom domain registration such as com.devore.widget are allowed?